deeployer.libsonnet

```diff
@@ -4,7 +4,6 @@
   local tag = namespace,
   local url = namespace+".test.workadventu.re",
   // develop branch does not use admin because of issue with SSL certificate of admin as of now.
-  local adminUrl = if namespace == "master" || namespace == "develop" || std.startsWith(namespace, "admin") then "https://"+url else null,
```

Comment:
It could still be useful to be able to test the connexion to the Admin, even from the Github CD.
But I agree, it makes little sense to test it in "master" or "develop" branch.
However, keeping the possibility in branches starting with "admin" could be useful in the future.

So what about simply doing this only branches starting with "admin" ?

```
local adminUrl = std.startsWith(namespace, "admin") then "https://"+url else null,
```

Reply:
This wouldn't work, the admin url would be admin.test.workadventu.re which doesn't exist, we need to hardcode admin.workadventu.re